### ISSUE TITLE
Default mailable design should be accessible

### ISSUE THREAD

**Issue by MikeHopley** [laravel/ideas#2227]
When Laravel sends an HTML email, the default body text fails WCAG contrast requirements. It should be made darker.

**Comment by JustSteveKing** [comment]
This is already available to customise as per your own requirements if you follow the docs. While I agree that the web should be accessible by design, sometimes you have to balance design against accessibility.

[https://laravel.com/docs/7.x/mail#customizing-the-components](Laravel docs to customise mailables)

**Comment by MikeHopley** [comment]
I know it can be customised, and have done so.

I am suggesting that Laravel set a good example and start with accessible text. The designer can just as well customise it back to light grey, if they are foolish enough.

**Comment by bjuppa** [comment]
I didn't see this issue before posting my own... sorry! I put actionable suggestions in this one: https://github.com/laravel/ideas/issues/2524
